docs/UserGuide.adoc

```diff
@@ -129,9 +129,25 @@ This command allows you to view a recipe, the ingredients required, and the step
 
 - Format: `cookbook view recipe INDEX`
 - Example:
-* `cookbook view recipe 1` +
+* `cookbook view 1` +
     Views the ingredients and steps of recipe 1.
 
+==== Add a new recipe to the cookbook
+This command allows you to add and store a new recipe with the specified recipe name to the cookbook.
+
+- Format: `cookbook add n/NAME`
+- Example:
+* `cookbook add n/Bacon Carbonara` +
+    Adds a new recipe with the name 'Bacon Carbonara'.
+
+==== Delete a recipe from the cookbook
+This command allows you to delete a recipe of the specified index from the cookbook.
+
+- Format: `cookbook delete INDEX`
```

Comment:
Are we standardising the command to "remove"?

Reply:
My bad. Will edit accordingly in the next commit.